### ISSUE TITLE
steps values in configs/pointpillars/xyres_16.config

### ISSUE THREAD

**Issue by FrancoisMasson1990** [traveller59/second.pytorch#300]
Hello,

In the config file configs/pointpillars/xyres_16.config, you define for exemple : 
steps: 296960 # 92800 # 1856 steps per epoch * 160 epochs
But how do you get these values ? (Where the number of epochs and the number of steps is defined)

**Comment by darshb34** [comment]
1.) number of steps per epoch = (number of training samples)/batch_size
                          = 3712/2 = 1856
     total steps for 160 epochs = number of steps per epoch * 160 = 1856 *160

     epochs = 160 (from the paper and also because the network converges after 160 epochs)
     number of training samples = 3712 (from the paper)
     batch size = 2 (from .config file)

2.) As quoted by the authhors in  "PointPillar" and even the "SECOND" paper,: "For experimental studies we split the official training into **3712** training samples and 3769 validation samples [1], while for our test submission we created a minival set of 784 samples from the validation set and trained on the remaining 6733 samples." Also, "To optimize the loss function we use the Adam optimizer with an initial learning rate of 2 ∗ 10−4 and decay the learning rate by a factor of 0:8 every 15 epochs and train for **160 epochs**. We use a batch size of 2 for validation set and 4 for
our test submission."

3.) In xyres_16.config, the batch size is 2 (you can increase this if you want faster training. If you increase this you have to decrease the steps).
